apps/mediator/src/pickup/postgresLoader.ts

```diff
@@ -1,4 +1,4 @@
-import { PostgresMessagePickupRepository } from '../../../../packages/message-pickup-repository-pg/src/PostgresMessagePickupRepository'
+import { TransportQueuePostgres } from '../../../../packages/transport-queue-postgres/src/TransportQueuePostgres'
```

Comment:
This won't really work with these imports right? Wouldn't it make sense to first merge #55, and then based on the new structure publish the packages to npm?

Reply:
From my side #55 is fully ready to merge, just wanted to get a review (which i now got from both of you!)